resources/measures/upgrade_hvac_economizer/tests/hvac_economizer_test.rb

```diff
@@ -171,11 +169,156 @@ def apply_measure_and_run(test_name, measure, argument_map, osm_path, epw_path,
+
+      # get related airloophvac
+      air_loop_hvac = sizing_system.airLoopHVAC
+      name_air_loop_hvac = air_loop_hvac.name.to_s
+
+      # get design OA flow rate
+      oa_design_rate = nil
+      if sizing_system.autosizedDesignOutdoorAirFlowRate.is_initialized
+        oa_design_rate = sizing_system.autosizedDesignOutdoorAirFlowRate.get
+      elsif sizing_system.designOutdoorAirFlowRate.is_initialized
+        oa_design_rate = sizing_system.designOutdoorAirFlowRate.get
+      else
+        raise 'no design OA flow rate found'
+      end
+      puts("### DEBUGGING: original | name_air_loop_hvac = #{name_air_loop_hvac} | oa_design_rate = #{oa_design_rate}")
+
+      # add key (airloop name) and value (design OA rate)
+      hash_oa_design_rates[name_air_loop_hvac] = oa_design_rate.round(6)
+
+    end
+
+    return hash_oa_design_rates
+  end
+
+  def economizer_available(model)
+    economizer_availability = []
+    model.getAirLoopHVACs.each do |air_loop_hvac|
+      # get airLoopHVACOutdoorAirSystem
+      oa_sys = air_loop_hvac.airLoopHVACOutdoorAirSystem
+      if oa_sys.is_initialized
+        oa_sys = oa_sys.get
+      else
+        OpenStudio.logFree(OpenStudio::Error, 'openstudio.prototype.Model', "#{air_loop.name} is required to have an economizer, but it has no OA system.")
+        next
+      end
+      # get controller:outdoorair
+      oa_control = oa_sys.getControllerOutdoorAir
+      # change/check settings: control type
+      if oa_control.getEconomizerControlType != 'NoEconomizer'
+        economizer_availability << true
+      else
+        economizer_availability << false
+      end
+    end
+    return economizer_availability
+  end
+
+  def models_to_test_design_oa_rates
+    test_sets = []
+    test_sets << { model: 'PVAV_gas_heat_electric_reheat_4A', weather: 'CA_LOS-ANGELES-DOWNTOWN-USC_722874S_16', result: 'Success' }
+    test_sets << { model: 'PSZ-AC_with_gas_coil_heat_3B', weather: 'CA_LOS-ANGELES-DOWNTOWN-USC_722874S_16', result: 'Success' }
+    test_sets << { model: '361_Warehouse_PVAV_2a', weather: 'CA_LOS-ANGELES-DOWNTOWN-USC_722874S_16', result: 'Success' }
+    test_sets << { model: 'LargeOffice_VAV_chiller_boiler', weather: 'CA_LOS-ANGELES-DOWNTOWN-USC_722874S_16', result: 'Success' }
+    test_sets << { model: 'LargeOffice_VAV_district_chw_hw', weather: 'CA_LOS-ANGELES-DOWNTOWN-USC_722874S_16', result: 'Success' }
+    test_sets << { model: 'Outpatient_VAV_chiller_PFP_boxes', weather: 'CA_LOS-ANGELES-DOWNTOWN-USC_722874S_16', result: 'Success' }
+    test_sets << { model: 'Retail_PVAV_gas_ht_elec_rht', weather: 'CA_LOS-ANGELES-DOWNTOWN-USC_722874S_16', result: 'Success' }
+    test_sets << { model: 'VAV_chiller_boiler_4A', weather: 'CA_LOS-ANGELES-DOWNTOWN-USC_722874S_16', result: 'Success' }
+    test_sets << { model: 'VAV_with_reheat_3B', weather: 'CA_LOS-ANGELES-DOWNTOWN-USC_722874S_16', result: 'Success' }
+    return test_sets
+  end
+
+  def test_design_oa_rates
```

Comment:
We don't expect the specified design outdoor air flow rate to be an issue that needs to be checked. Something that we do need to make sure we verify is that the "Requested Mechanical Ventilation..." variable always represents the design outdoor air flow rate in multizone VAV systems, since we actuate to this value when we override economizer operation.

Reply:
added a new test `test_requested_oa_rates`